tests/unit/reco_utils/evaluation/test_spark_evaluation_diversity_metrics.py

```diff
@@ -0,0 +1,134 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import itertools
+import pytest
+import numpy as np
+import pandas as pd
+from pandas.util.testing import assert_frame_equal
+from pyspark.sql import Row
+from reco_utils.evaluation.diversity_evaluator import DiversityEvaluator
```

Comment:
you may need to wrap these imports to avoid problems when test collection scans this file in non-spark environments
```
try:
    from pyspark.sql import Row
    from reco_utils.evaluation.diversity_evaluator import DiversityEvaluator
except ImportError:    
    pass  # skip this import if we are in pure python environment
```

Reply:
addressed